core/src/main/kotlin/xtdb/util/TemporalBounds.kt

```diff
@@ -27,8 +27,11 @@ data class TemporalDimension(
 
 data class TemporalBounds(
     val validTime: TemporalDimension = TemporalDimension(),
-    val systemTime: TemporalDimension = TemporalDimension()
+    val systemTime: TemporalDimension = TemporalDimension(),
+    val maxSystemFrom: Long = Long.MAX_VALUE,
```

Comment:
This is just to make things work for now. We likely want a different abstraction here if `maxSystemFrom` is important.